### ISSUE TITLE
[BUG] visgui error opening events - KeyError: 'ShiftMeasure'

### ISSUE THREAD

**Issue by csoeller** [python-microscopy/python-microscopy#944]
**Describe the bug**
Opening h5r files with events in visgui leads to error and empty events display.

**To Reproduce**
1. Start visgui
2. Open https://github.com/csoeller/PYME-extra-sample-data/blob/master/data/MT-DNA-PAINT/19_11_series_C.h5r
3. Go to `Events` tab

Note this is with py37 build, everything works fine in my py27 build.

**Expected behavior**
My still existing py27 install displays the events just fine.

**Error raised**

```
Traceback (most recent call last):
  File "/Users/csoe002/Documents/src/PYME-src/pyme-py37/PYME/DSView/eventLogViewer.py", line 302, in OnPaint
    self.DoPaint(MemDC);
  File "/Users/csoe002/Documents/src/PYME-src/pyme-py37/PYME/DSView/eventLogViewer.py", line 230, in DoPaint
    dc.SetTextForeground(self.lineColours[sourceEv])
KeyError: 'ShiftMeasure'
Traceback (most recent call last):
  File "/Users/csoe002/Documents/src/PYME-src/pyme-py37/PYME/DSView/eventLogViewer.py", line 302, in OnPaint
    self.DoPaint(MemDC);
  File "/Users/csoe002/Documents/src/PYME-src/pyme-py37/PYME/DSView/eventLogViewer.py", line 230, in DoPaint
    dc.SetTextForeground(self.lineColours[sourceEv])
KeyError: 'ShiftMeasure'
```

**Platform (please complete the following information - or copy-paste from error dialog if shown):**
 - OS: macos 10.15.x
 - Python version: 3.7
 - numpy version: ??
 - wxpython version [if a UI bug]: 4.0.4

**Comment by David-Baddeley** [comment]
This is almost certainly a py3 bytes/string issue. Will have a quick look.

**Comment by David-Baddeley** [comment]
should be fixed in #945
